src/Main.js

```diff
@@ -9,12 +9,28 @@ class Main extends React.Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData.movies,
+      movieData: [],
       isExtendedView: false,
-      clickedPosterID: null
+      clickedPosterID: null,
+      loading: false
     }
   }
 
+  componentDidMount() {
+    console.log("did mount")
```

Comment:
We will want to remove the console logs down the road. They are extremely helpful for making sure everything is working though!